openapi/openapi.yml

```diff
@@ -56,14 +56,16 @@ paths:
           content:
             application/json:
               schema:
-                allOf:
-                  - $ref: '#/components/schemas/Success'
-                  - type: object
-                    properties:
-                      settings:
-                        $ref: '#/components/schemas/Settings'
-                      spocs:
-                        $ref: "#/components/schemas/SpocFeed"
```

Comment:
the "spocs" field is included based on the placement request by the client, so it could really be anything. this is handled by the switch to additionalProperties with the same $ref